OnlinerTask/OnlinerTask.Data/Notifications/Technologies/Implementations/NetMqNotificator.cs

```diff
@@ -25,15 +25,21 @@ private static void SendTcpMessage(string text, string tcpString, string type, s
             using (var client = NetMQContext.Create().CreateRequestSocket())
             {
                 client.Connect(tcpString);
```

Comment:
Correct. so common logic should be grouped. Lets create message and then configure client and send this message.